src/core/src/package_managers/TdnfPackageManager.py

```diff
@@ -80,7 +76,8 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.STR_TOTAL_DOWNLOAD_SIZE = "Total download size: "
         self.version_comparator = VersionComparator()
 
-        # if an Auto Patching request comes in on a Azure Linux machine with Security and/or Critical classifications selected, we need to install all patches, since classifications aren't available in Azure Linux repository
+        # Todo: Q: Do we need this now that get_security_updates() returns all updates?
```

Comment:
Marking todo comment spot for PR evolution.

Reply:
Behavior must be identical to CentOS - only default patching calls can get autoupgraded to Crit+Sec+Other. And sec updates returns everything. For any other type of call, the customer must explicitly specify everything or the request should fail.

Reply:
Modifed